### ISSUE TITLE
[PS2] Color palette issues in many games and Save state issues

### ISSUE THREAD

**Issue by Sin2FX** [libretro/snes9x2002#46]
In many games the palhete and sometimes the entire screen becomes yellowish. This is very evident right at the beginning of Super Metroid but happens in many others games such as the menu in F-zero, The Lost Vikings. Sometimes the transparency effects becomes yellowish like the clouds on the second stage of Donkey Kong Country 2. Another problem is save states it generally get stuck at 78% saving

**Comment by keithbowes** [comment]
Yeah, this is a very old version and has so many glitches.  Super Mario Kart (my most played SNES game) is unplayable.  I suppose it's fine if you have really low-end hardware and don't mind some games not working correctly or at all, but really, if possible, you should use Snes9x 2005 or later (or maybe try Supafaust, which is also for low-end hardware).
